terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,7 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+# Scheduled scaling of tasks
+service_autoscale_enabled  = true
+service_scaledown_schedule = "55 19 * * ? *"
```

Comment:
Can we remove the service_scaledown_schedule & service_scaleup_schedule from cidev environment?